allennlp/modules/transformer/self_attention.py

```diff
@@ -45,6 +58,10 @@ def __init__(
         else:
             self.attn = Attention.by_name(self.scoring_func)()
 
+        # out linear layer for distilbert.
+        if output_linear:
+            self.output = torch.nn.Linear(hidden_size, self.all_head_size)
```

Comment:
What is this? Does it get initialized as well?

Reply:
Distilbert has this extra linear layer at the end. Trying to ensure that the module works for different variants.